src/scanner.rs

```diff
@@ -58,6 +58,9 @@ impl FileSet {
             glob_path.clone()
         };
 
+        // Parse codeowners.
+        let codeowners = codeowners::locate(".").map(|path| codeowners::from_path(path.as_path()));
```

Comment:
It might be better to have a list of possible directories for the `CODEOWNERS` file (e.g., `.github/`) instead of searching through everything - not great for big repos

Reply:
For GitHub CODEOWNERS file location: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location
Bitbucket: https://confluence.atlassian.com/bitbucketserver/code-owners-1296171116.html
GitLab: https://docs.gitlab.com/ee/user/project/codeowners/#codeowners-file

Given the docs above, we could first try searching for the `CODEOWNERS` file in a list of directories like `.github/`, `.bitbucket/`, `.gitlab/`, `docs/` and only searching in `"."` if we haven't found a file as yet.

Reply:
Let's look in those 4 directories and have a manual override if you are doing something else. I want to avoid scanning the file system as much as possible.